projects/dnd/src/directives/dnd-item.directive.ts

```diff
@@ -29,17 +42,22 @@ export class DndItemDirective<T> implements OnChanges, OnInit, OnDestroy {
     this.processDndItemIdAndDataChanges(changes);
   }
 
-  public ngOnInit(): void {
+  public ngAfterViewInit(): void {
     this.renderer.setAttribute(this.elementRef.nativeElement, DND_ITEM_ID_ATTRIBUTE, this.dndItemId);
 
-    const dndItemConfig: DndItemConfig | undefined = this.dndItemRegistryService.getDndItemConfig(this.dndItemId);
+    let dndItemConfig: DndItemConfig | undefined = this.dndItemRegistryService.getDndItemConfig(this.dndItemId);
```

Comment:
Вроде так выглядит приятнее

```suggestion
    const dndItemConfig: DndItemConfig | undefined = this.dndItemRegistryService.getDndItemConfig(this.dndItemId) ?? {
        dndItem: { id: this.dndItemId, data: this.dndItemData, elementParts: [this.elementRef.nativeElement] },
        itemTemplate: this.dndItemTemplateRef,
        canBeMoved: this.canBeMoved,
        canBeDroppableFor: this.canBeDroppableFor,
      };
```

Reply:
Так красиво не получится, потому что там есть один участок кода, который в проверке должен выполняться. Немного иначе переписал, вроде норм